scm/scm.go

```diff
@@ -103,6 +99,19 @@ type NewCourseOptions struct {
 	CourseCreator  string
 }
 
+// UpdateEnrollmentOptions contain information about enrollment.
+type UpdateEnrollmentOptions struct {
```

Comment:
If the `isValid()` method is close to the struct definition, then it is easier to maintain ... making sure that you remember to update the `isValid()` method if you add a field to the struct.

Reply:
What should I do then? Should I remove the `valid()` method for `UpdateEnrollmentOptions` or should I move it here? Should I do the same for all the other `valid()` methods? 

Should we create an issue and do it for all the `valid()` methods in the scm package in a separate PR?

Reply:
Let's create an issue to do it in a separate PR.